docs/user-guide/federation/deployment.md

```diff
@@ -0,0 +1,104 @@
+---
+---
+
+This guide explains how to use Deployments in the Federation control plane.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
```

Comment:
Please see the comments I made on the Prerequisites section here: https://github.com/kubernetes/kubernetes.github.io/pull/1827
and apply them as necessary to this document.